main.js

```diff
@@ -211,6 +220,44 @@ function createJitsiMeetWindow() {
     mainWindow.once('ready-to-show', () => {
         mainWindow.show();
     });
+
+    /**
+     * This is for windows [win32]
+     * so when someone tries to enter something like jitsi-meet://test
+     *  while app is closed
+     * it will trigger this event below
+     */
+    if (process.platform === 'win32') {
+        handleProtocolCall(process.argv.pop());
+    }
+}
+
+/**
+ * Handler for application protocol links to initiate a conference.
+ */
+function handleProtocolCall(fullProtocolCall) {
+    // don't touch when something is bad
+    if (
+        !fullProtocolCall
+        || fullProtocolCall.trim() === ''
```

Comment:
this second test is unnecessary, isn't it?

Reply:
Well maybe in some cases `fullProtocolCall` is defined but is filled by spaces. Looks also strange